kahuna/public/js/components/gr-permissions-filter/gr-permissions-filter.tsx

```diff
@@ -0,0 +1,236 @@
+import * as React from "react";
+import * as angular from "angular";
+import { react2angular } from "react2angular";
+import { useEffect, useRef, useState, KeyboardEvent } from "react";
+import * as PermissionsConf from "./gr-permissions-filter-config";
+
+import "./gr-permissions-filter.css";
+import "./gr-toggle-switch.css";
+
+const SHOW_CHARGEABLE = "Show payable images";
+const SHOW_CHARGEABLE_SHORT = "Payable";
+const SELECT_OPTION = "Select an option";
+const CONTROL_TITLE = "Permissions Selector";
+const SELECTED = "Selected";
+const NOT_SELECTED = " Not Selected";
```

Comment:
```suggestion
const NOT_SELECTED = "Not Selected";
```

Reply:
Will include this change in next related PR